apps/qubit/config/qubitConfiguration.class.php

```diff
@@ -169,5 +170,17 @@ protected function bootstrapDrmc()
     sfConfig::set('app_drmc_ss_user', $config['ARCHIVEMATICA_SS_USER']);
     sfConfig::set('app_drmc_ss_api_key', $config['ARCHIVEMATICA_SS_API_KEY']);
     sfConfig::set('app_drmc_tms_url', $config['ATOM_DRMC_TMS_URL']);
+
+    // This are not real TMS API versions, just a way to differentiate
+    // between the two current implementations.
+    $allowedTmsVersions = array('1', '2');
```

Comment:
I didn't want to use the client names in here, but I'm open to better suggestions than "1" and "2" ;)

Reply:
I think this is fine.  The important bit is that we have a way to distinguish between them, and can iterate on the versions.